<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createTask</class>
 <widget class="QWidget" name="createTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание задачи</string>
  </property>
  <widget class="QLabel" name="label_info">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Создание задачи в проекте: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Заголовок:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_title">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_description">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Описание:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_description">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Статус задачи: </string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget_select_date">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>251</width>
     <height>173</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_assignee_to">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Назначить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_deadline">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Дедлайн:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_select_assignee">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>251</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_create_task">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать задачу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open_calendar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Календарь</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
